src/Backend/OnboardingInsuranceAPI/Areas/User/UserController.cs

```diff
@@ -21,7 +21,7 @@ public UserController(GetUser getUser, RegisterUser registerUser , UpdateUser up
 
     [Function("GetUser")] //read user profile by PID number
     public  async Task<HttpResponseData> Get(
-        [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "users")] HttpRequestData req) //TODO rewrite frontend remove pid and fix reference 
+        [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "user")] HttpRequestData req) //TODO rewrite frontend remove pid and fix reference 
```

Comment:
Why change it to "user"? We have talked about this before. You are accessing "recourse" of some kind in this case you are accessing a complete list of "users" and then you are filtering in them by the pid in JWT

Reply:
Same thing with PUT

Reply:
Do you understand what im saying? You want to do kind of CRUD operation on "users" resource. Which includes GET/Read.
By saying "user" you specifying one user which is not a resource. If you wanted that kind of router it should something like: "/me" "/current-user" or something.

Reply:
https://restfulapi.net/resource-naming/

Reply:
Ok, yeah now I get it.